src/V3AstNodes.cpp

```diff
@@ -1073,6 +1073,33 @@ string AstBasicDType::prettyDTypeName() const {
 void AstNodeMath::dump(std::ostream& str) const { this->AstNode::dump(str); }
 void AstNodeUniop::dump(std::ostream& str) const { this->AstNodeMath::dump(str); }
 
+AstNodeMath* AstStdRandomize::newStdRandomize(FileLine* fl, AstNodeVarRef* varp, int offset,
+                                              AstMemberDType* memberp) {
```

Comment:
This might be better in the visitor rather than as a node function - unless used multiple places.